GetF!t/Modules/Exercises/ExercisesView.swift

```diff
@@ -73,10 +69,8 @@ struct ExerciseView: View {
                                 .environmentObject(history)
                         case .timer:
                             TimerView(timerViewModel: TimerViewModel.init(exerciseName: Exercise.exercises[index].exerciseName))
```

Comment:
Nevyrieši sa ten problém s timer view keď dáš preč inicializáciu tohto view modelu z body property? ObservableObjecty by si nikdy nemal incializovat v konštruktore alebo v body nejakého view. Vždy ho musíš inicializovat ako property nejakého view s tým že použiješ @StateObject wrapper.
Budeš mat problém že pri vytváraní properties ešte nebudeš mat prístup k indexu a tak nebudeš vedieť aké máš predať exercise name. To môžeš vyriešiť tak že ho predáš viewmodelu na onAppear tohto exercise view.